support/modules/rootfs-image

```diff
@@ -5,10 +5,27 @@ set -ue
 STATE="$1"
 FILES="$2"
 
+if [ -n "$(which jq >/dev/null 2>&1)" ]; then
+    JQ_AVAILABLE=1
+else
+    JQ_AVAILABLE=0
+fi
+
 CONF_FILE="/var/lib/mender/mender.conf"
 
-MENDER_ROOTFS_PART_A="$(jq -r .RootfsPartA < "$CONF_FILE")"
-MENDER_ROOTFS_PART_B="$(jq -r .RootfsPartB < "$CONF_FILE")"
+if [ "$JQ_AVAILABLE" = 1 ]; then
+    MENDER_ROOTFS_PART_A="$(jq -r .RootfsPartA < "$CONF_FILE")"
+    MENDER_ROOTFS_PART_B="$(jq -r .RootfsPartB < "$CONF_FILE")"
+else
+    # Fall back to line based parsing. Vulnerable to weird JSON nesting, as well as unexpected
+    # newlines, although it is unlikely with a regular configuration file.
+
+    # Poor man's case insensitive match.
```

Comment:
:laughing:

Reply:
Is it needed though? Our format is specified? Client won't accept but one format here?

Reply:
I can't find if the JSON standard itself is case insensitive, but at least [the Golang parser is](https://pkg.go.dev/encoding/json#Unmarshal).

Reply:
`JSON` is case sensitive, coming from Ecma

Reply:
Hmm ok. I think we should probably do whatever Golang does though, unless you think it's a bad idea.

Reply:
Agree